From c11dcbb0dd70f9a824c85f0947d7083adab8ea1f Mon Sep 17 00:00:00 2001
From: Lei Xu <lei.xu@nxp.com>
Date: Wed, 3 Sep 2025 10:41:14 +0800
Subject: [PATCH 51/56] LF-16159: arm64: dts: imx95-15x15-frdm: update mclk-fs

Update mclk-fs value to avoid below warning:
fsl-mqs mqs1: can't get proper divider

Signed-off-by: Lei Xu <lei.xu@nxp.com>
Reviewed-by: Shengjiu Wang <shengjiu.wang@nxp.com>
Acked-by: Jason Liu <jason.hui.liu@nxp.com>
---
 arch/arm64/boot/dts/freescale/imx95-15x15-frdm.dts | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm64/boot/dts/freescale/imx95-15x15-frdm.dts b/arch/arm64/boot/dts/freescale/imx95-15x15-frdm.dts
index 230fb4ed605b..6e8a87e5b4ae 100644
--- a/arch/arm64/boot/dts/freescale/imx95-15x15-frdm.dts
+++ b/arch/arm64/boot/dts/freescale/imx95-15x15-frdm.dts
@@ -1101,7 +1101,7 @@ sai1_port1: port@1 {
 			bitclock-master;
 			frame-master;
 			playback-only;
-			mclk-fs = <256>;
+			mclk-fs = <512>;
 
 			sai1_endpoint1: endpoint {
 				remote-endpoint = <&mqs1_endpoint>;
-- 
2.34.1

